manual.md

```diff
@@ -686,17 +703,18 @@ purpose functions supported by
 FastBasic. Shown are the full syntax
 and the abbreviated syntax.
 
-- TIME / T. :
+- TIME() / T. :
```

Comment:
`TIME` is special, it does not need the parentheses.

Reply:
Removed () and added a note making it clear that TIME is special, and does not use parens. (In https://github.com/dmsc/fastbasic/pull/59/commits/350fd46637313c45409f17825638d5d13b4996a3)

Thanks!